bears/general/InvalidLinkBear.py

```diff
@@ -1,7 +1,9 @@
 import re
 import requests
+from urllib.parse import urlparse
```

Comment:
This was added to use `urlparse` to find the host of the given url. And then filter those urls with 
```py
if host in link:
    # the timeout for this link will be the one mentioned in `.coafile`
```

But ATM I'm not able to find how to assert the found URL with a given timeout. For that I'll have to change the function `find_links_in_file`. Suggestions for implementing this are welcome.